src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -200,7 +200,7 @@
             "MaxPathsPerInvoke"
         ],
         "Bridged Device Basic Information": ["ProductAppearance"],
-        "Chime": ["ActiveChimeID", "Enabled"],
+        "Chime": ["Enabled"],
```

Comment:
Why this change?

Reply:
Alchemy did this because there is no ActiveChimeID attribute in this cluster. There's a SelectedChime attribute, which is probably what this was meant to be?

Reply:
Reverted changes to the zcl.json and zcl-with-test-extensions.json since the change is not related to the current cluster.

Reply:
Sure, but there isn't in the spec. 

I'm updating Alchemy to not touch parts of this file that weren't passed as arguments on the command line, so this shouldn't happen again, but this is revealing a mismatch between the spec and the SDK.